proxy/agent.go

```diff
@@ -164,9 +164,9 @@ func (a *agent) Run(stop <-chan struct{}) {
 			// schedule a retry for a transient error
 			if status.err != nil && !reflect.DeepEqual(a.desired, a.current) {
 				if a.retryBudget > 0 {
-					delay = a.retryInterval * (2 << uint(a.maxRetries-a.retryBudget))
+					delay = a.retryInterval * (1 << uint(a.maxRetries-a.retryBudget))
```

Comment:
Initialize retryBudget with maxRetries when agent is initialized, there is no guarantee case config is handled before case status.